stdlib/os/__init__.pyi

```diff
@@ -822,6 +822,34 @@ else:
 
 if sys.platform != "win32":
     from posix import sched_param
+
+    if sys.version_info >= (3, 8):
+        def posix_spawn(
+            path: _FdOrAnyPath,
+            argv: _ExecVArgs,
+            env: _ExecEnv,
+            *,
+            file_actions: Optional[Sequence[Tuple[int, Any]]] = ...,
+            setpgroup: Optional[int] = ...,
+            resetids: bool = ...,
+            setsid: bool = ...,
+            setsigmask: Tuple[Any, ...] = ...,
+            setsigdef: Tuple[Any, ...] = ...,
```

Comment:
```suggestion
            setsigmask: Iterable[int] = ...,
            setsigdef: Iterable[int] = ...,
```
From https://github.com/python/cpython/blob/main/Modules/posixmodule.c#L1505 it looks like any Iterable[int] works.